test/test_functional_tensor.py

```diff
@@ -14,7 +14,8 @@
 import torchvision.transforms as T
 from torchvision.transforms import InterpolationMode
 
-from common_utils import TransformsTester, cpu_and_gpu, needs_cuda
+from common_utils import (cpu_and_gpu, needs_cuda, _create_data, _create_data_batch, _assert_equal_tensor_to_pil,
```

Comment:
Something like this may be better:
```
from common_utils import (
    cpu_and_gpu,
    needs_cuda,
    _create_data,
    _create_data_batch,
   _assert_equal_tensor_to_pil,
```